services/app/app/components/directory/category-tree.js

```diff
@@ -1,9 +1,10 @@
 import Component from '@ember/component';
 import { computed } from '@ember/object';
+import ActionMixin from '@base-cms/company-update-app/mixins/action';
 
 const { error } = console;
 
-export default Component.extend({
+export default Component.extend(ActionMixin, {
```

Comment:
This component doesn't do any loading state manipulation -- why is this mixin being included?